judo-runtime-core-jsl-itest/src/test/java/hu/blackbelt/judo/runtime/core/jsl/transfer/MappedTransferFunctionsTest.java

```diff
@@ -0,0 +1,1023 @@
+
+        assertEquals(Optional.of(34L), collectionFunctions.getDivisionConst());
+        assertEquals(Optional.of(35L), collectionFunctions.getRoundConst());
+
+        assertEquals(2, transferCollectionFunctionsDao.queryFirstChildrenField(collectionFunctions).execute().size());
+        assertEquals(2, transferCollectionFunctionsDao.queryFirstChildrenRelation(collectionFunctions).execute().size());
+
+        assertEquals(2, transferCollectionFunctionsDao.queryLastChildrenField(collectionFunctions).execute().size());
+        assertEquals(2, transferCollectionFunctionsDao.queryLastChildrenRelation(collectionFunctions).execute().size());
+
+        assertEquals(2, transferCollectionFunctionsDao.queryFrontChildrenField(collectionFunctions).execute().size());
+        assertEquals(2, transferCollectionFunctionsDao.queryFrontChildrenRelation(collectionFunctions).execute().size());
+
+        assertEquals(2, transferCollectionFunctionsDao.queryBackChildrenField(collectionFunctions).execute().size());
+        assertEquals(2, transferCollectionFunctionsDao.queryBackChildrenRelation(collectionFunctions).execute().size());
+
+        assertTrue(collectionFunctions.getContainsParent().orElseThrow());
+
+        assertEquals(1, transferCollectionFunctionsDao.queryFilterParentsField(collectionFunctions).execute().size());
+        assertEquals(1, transferCollectionFunctionsDao.queryFilterParentsRelation(collectionFunctions).execute().size());
+
+        assertEquals(1, transferCollectionFunctionsDao.queryFilterChildrenField(collectionFunctions).execute().size());
+        assertEquals(1, transferCollectionFunctionsDao.queryFilterChildrenRelation(collectionFunctions).execute().size());
+
+        assertEquals(Optional.of(true), collectionFunctions.getAnyTrueChildrenField());
+        assertEquals(Optional.of(false), collectionFunctions.getAnyTrueChildrenRelation());
+
+        assertEquals(Optional.of(true), collectionFunctions.getAllTrueChildrenField());
+        assertEquals(Optional.of(false), collectionFunctions.getAllTrueChildrenRelation());
+
+        assertEquals(Optional.of(false), collectionFunctions.getAnyFalseChildrenField());
+        assertEquals(Optional.of(true), collectionFunctions.getAnyFalseChildrenRelation());
+
+        assertEquals(Optional.of(false), collectionFunctions.getAllFalseChildrenField());
+        assertEquals(Optional.of(false), collectionFunctions.getAllFalseChildrenRelation());
+    }
+
+    @Test
+    @Requirement(reqs = {
+            "REQ-EXPR-022",
+            "REQ-SRV-002"
+    })
+    public void testBooleanAggregatorFunctionsOnMappedTransfer() {
+        /*
+
+        |--------------------------------------------|
+        | Transfer || 3 | 2 | 2 | 1 | 1 | 1 | 0 | 0 | 0 | 0 |
+        | F || 0 | 1 | 0 | 2 | 1 | 0 | 3 | 2 | 1 | 0 |
+        | U || 0 | 0 | 1 | 0 | 1 | 2 | 0 | 1 | 2 | 3 |
+        |--------------------------------------------|
```

Comment:
Pattern is "broken" with "Transfer":D

Reply:
Then is broken in the entity Function test also :D